db/migrate/20210526091948_add_column_to_hall.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToHall < ActiveRecord::Migration[6.1]
+  def change
+    add_column :halls, :hall_number, :integer
```

Comment:
I don't know the business needs of the project, but you can consider the nullability of this field. By default, all new columns are `nil` by default and optional.